extension/js/common/api/email-provider/gmail/google-auth.ts

```diff
@@ -129,14 +129,9 @@ export class GoogleAuth {
       GoogleAuth.waitForAndProcessOauthWindowResult(oauthWin.id, acctEmail, scopes, authRequest.csrfToken, save),
       GoogleAuth.waitForOauthWindowClosed(oauthWin.id, acctEmail),
     ]);
-    try {
-      chrome.windows.remove(oauthWin.id);
-    } catch (e) {
-      if (String(e).indexOf('No window with id') === -1) {
-        Catch.reportErr(e);
-      }
```

Comment:
Why do you propose removing the try/catch? Did it give you any trouble?

Reply:
I figured out the error was with `chrome.windows.remove(oauthWin.id);` method. On cancel the chrome window is already removed by itself and we don't have the windows, so it throws the error. What we need to do was to delete the windows once the result is returned from the promise. That way we don't need the try/catch.

Reply:
If you want me to fix it by the suggested way I can do it as well.

Reply:
Ok. That does make sense. We can go with your proposal then, but I'll need to fix a failing test - there's a missing await. Take a look. https://flowcrypt.semaphoreci.com/workflows/44c0d9e4-4bc3-49e0-8ebc-9a8ce0c337d1